libs/core/src/utils/utils.spec.ts

```diff
@@ -17,3 +20,9 @@ describe('format', () => {
     expect(format('Joseph', 'Quincy', 'Publique')).toEqual('Joseph Quincy Publique');
   });
 });
+
+describe('isRequired', () => {
```

Comment:
Note to @QuintonJason Make better tests in a followup PR. Checking for validity is dependent on the element type which is different per form element.

Reply:
It may be better to add a comment in the code, so that there is a reminder.

```
//TODO: Make better test for isRequired function
```